### PR TITLE
Fix wrong include and shared libray paths

### PR DESCRIPTION
**Pull request by omichel**
This is some clean-up.

### CODE REVIEW

**Comment by omichel** [comment]
👍

**Comment by fabienrohrer** [comment]
Just a question; I'm wondering how this could work outside an msys environment, i.e. when running Webots from the icon. Did you test this?

**Comment by omichel** [comment]
Yes, it works as MSYS2 considers that its root folder is `C:\Program Files\Webots\msys64`.

**Comment by DavidMansolino** [comment]
I am currently creating a package to test this (ant other PRs).

File: projects/samples/howto/controllers/vision/Makefile [hunk 1]
```diff
@@ -27,7 +27,7 @@ USE_C_API = true
 ifeq ($(OSTYPE),windows)
   OPENCV_VERSION := $(subst .,,$(shell opencv_version))
   INCLUDE = -I/mingw64/include/opencv4
-  LIBRARIES = -L"$(WEBOTS_HOME)/msys64/mingw64/bin" -L/mingw64/bin -lopencv_core$(OPENCV_VERSION) -lopencv_imgproc$(OPENCV_VERSION)
+  LIBRARIES = -L/mingw64/bin -lopencv_core$(OPENCV_VERSION) -lopencv_imgproc$(OPENCV_VERSION)
```
Comment (DavidMansolino) [inline projects/samples/howto/controllers/vision/Makefile]: I can't neither run or recompile this controller from the release (both works fine from the development environement):
```
make
make: opencv_version: Command not found
# updating vision.d
make: opencv_version: Command not found
# compiling vision.cpp
vision.cpp:25:10: fatal error: opencv2/core/core.hpp: No such file or directory
 #include <opencv2/core/core.hpp>
          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [C:/Program Files/Webots/resources/Makefile.include:691: build/release/vision.o] Error 1
```
capture

**Comment by omichel** [comment]
I believe the errors you got are not related with the changes in the Makefile and are not a regression. I just added a comment in the vision.cpp file to explain how to proceed to use this controller. Distributing the OpenCV dependency on Windows is the only way to fix this problem, but that would add an extra 100MB+ to the package, which seems overkill for just one controller example. So, I would propose to keep things like this.

👍

**Comment by DavidMansolino** [comment]
I fixed a few others paths I will create a package to make sure it works fine.

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Pull request merged**